docs/reference/rosbridge.md

```diff
@@ -78,6 +78,46 @@ Parameters:
 
 - **message**: *engine.target.Pose*  
   OpenDR pose to be converted to ROS pose.
+ 
+#### `ROSBridge.from_ros_face`
+
+```python
+ROSBridge.from_ros_face(self,
+                        ros_hypothesis)
+```
+
+Converts a ROS ObjectHypothesis into an OpenDR Category.
+
+Parameters:
+
+- **message**: *ros_bridge.msg.Pose*  
+  ROS pose to be converted into an OpenDR pose.
```

Comment:
Needs to be updated

Reply:
Updated it now.

> I saw you resolved the conversation, but there's no change to the md file. Is it correct that the `to_ros_face` takes a pose message as parameter? Isn't that `ObjectHypothesisWithPose` instead of `ObjectHypothesis` ?

ObjectHypothesis is correct for fr.